locksmith/package.json

```diff
@@ -8,7 +8,7 @@
     "prod": "tsx -r ./config/envLoader.js ./src/server.ts",
     "start": "./scripts/start.sh server",
     "runner": "./scripts/start.sh runner",
-    "dev": "NODE_ENV=development yarn db:migrate && tsx watch -r ./config/envLoader.js src/server.ts",
+    "dev": "NODE_ENV=development && yarn db:migrate && tsx watch -r ./config/envLoader.js src/server.ts",
```

Comment:
That won't work. If you do this, then NODE_ENV will *not* be `development` when running `yarn db:migrate`